src/k8s/pkg/k8sd/types/cluster_config.go

```diff
@@ -49,10 +49,15 @@ type K8sDqlite struct {
 	Port int `yaml:"port,omitempty"`
 }
 
+func (c *ClusterConfig) SetDefaults() {
```

Comment:
I guess this is a PoC implementation and we eventually move `DefaultClusterConfig` into `SetDefaults`?